settings/src/main/java/org/odk/collect/settings/keys/ProjectKeys.kt

```diff
@@ -9,10 +9,6 @@ object ProjectKeys {
     const val KEY_USERNAME = "username"
     const val KEY_PASSWORD = "password"
 
-    // google_preferences.xml
-    const val KEY_SELECTED_GOOGLE_ACCOUNT = "selected_google_account"
```

Comment:
Are we able to get rid of `KEY_PROTOCOL` as well?

Reply:
We still need it in a few places:
- When importing settings. Then if a QR code contains protocol and it's GD we do not allow importing.
- When upgrading the version to detect old GD projects and convert them.

Reply:
Ah interesting!